# Homework 6

#### Note: to clear the environment in R, do:
`rm(list=ls(all=TRUE))`

## 0. Explain the difference between hypothesis testing and confidence intervals?

Before we note the difference between hypothesis testing and confidence intervals we have to mention the similarities between both which is that both are inferential methods that approximate sampling distributions using the data from a sample. The mean difference between the two methods is that hypothesis testing main goals is to relate a single conclusion of statistical significance against no statistical significance while confidence intervals main goal is to provide a range of probable values for your population. Hypothesis testng is commonly used when we want to do a precise comparison with predefined hypothesized parameter and significance level while confidence interval are used to describe a single sample.

## 1. A cellphone manufacturer claims the mean life of their “super-Li” battery is 500 days with a standard deviation of 25 days. You bought this cellphone and now you think this claim is inflated. As a conscious customer, you would like to test your belief and prove this manufacturer wrong, so that you and others can get part of the money spent on this device back.

### a. What is your variable?

### b. What statistic will you use to study this variable?

### c. What is your hypothesis? What is the null hypothesis?

### d. State the assumptions for your hypothesis test.

### e. What kind of a sample will you collect for an un-biased design?

### f. What will be your sample size?

a) My variable is the life expectancy of the battery in days
\
b) Z statistics since we are given the standard deviation of the population
\
c)
$
H_0:\mu \geq 500
\\
H_a:\mu<500
$
\
d) We are assuming that data is normally distributed or follows the central limit theorem
\
e) For this problem the best option is to obtain systematic random samples
\
f) We are going to calculate the sample size at a 95 % confidence interval and a margin of error of 2, we are going to choose the standard deviation to be 25
\
$
n=\frac{1.96^2\sigma ^2}{E^2}
\\
n=\frac{(1.96^2)(25 ^2)}{2^2}=600
$
\
Therefore we need around 600 devices to see any valueable data

## 2. Power and probability of type II error calculation: Consider two populations with $n = 50$: $\mathcal{N_1}(50, 100)$ and $\mathcal{N_2}(55, 100)$. Following steps will guide you to calculate type I, type II errors and power of a test.

### a. Let $X1$ denote the cutoff point for accepting $H0: \mu = 50$. For $\alpha = 0.05$, find the value of $X1$. 
Use the $z$-table given normal and population standard deviation. Values of $X$ greater than $X1$ will occur $5$% of the time when $H0$ is true.

### b. What is $H0$ were to be false and $H1: \mu = 55$ were to be true? Now any value below the $X1$ value that you calculated in (a) would be viewed as supporting $H0$. So find $P(X < X1)$. Again, use the $Z$-table. What does this probability suggest? This probability is $\beta$. What is the power $(1 - \beta)$?

### c. Repeat the same steps for $\alpha = 0.01$ and $\alpha = 0.001$.


# Part A
First step is to set the hypothesis
\
$
H_0: \mu =50
\\
H_a: \mu > 50
$
\
In this question we are going to use a z-test
\
$
z = \frac{\sqrt{n}(\bar{x} - \mu_0)}{\sigma} = \frac{\sqrt{50}(55-50)}{10} = 3.535534$
\
Now we need to find the critical values
\
$
z_{1-\alpha} = z_{0.95} = 1.644853$
\
Is $z > z_{0.95}$? Yes
\
Therefore we reject the null hypothesis
# Part B
$
\bar{x}=\mu + z_{1-alpha}(\frac{\sigma}{\sqrt{n}})
\\
\bar{x}=50 + (1.645)(\frac{10}{\sqrt{50}})
\\
\bar{x}=52.32638
$
Now we calculate the type II error
\
$
z=\frac{\bar{x} - \mu}{\frac{\sigma}{\sqrt{n}}}
\\
z=\frac{52.32638 - 55}{\frac{10}{\sqrt{50}}}
\\
z=−1.8905
\\
$
\
Now we find the p-value of that z-score
\
1-0.0293455=0.97065444
\
This is the power

# Part C
# $\alpha=0.01$
\
First step is to set the hypothesis
\
$
H_0: \mu =50
\\
H_a: \mu > 50
$
\
In this question we are going to use a z-test
\
$
z = \frac{\sqrt{n}(\bar{x} - \mu_0)}{\sigma} = \frac{\sqrt{50}(55-50)}{10} = 3.535534$
\
Now we need to find the critical values
\
$
z_{1-\alpha} = z_{0.99} = 2.32634787$
\
Is $z > z_{0.99}$? Yes
\
Therefore we reject the null hypothesis
\
Now we are going to calculate the power
\
$
\bar{x}=\mu + z_{1-alpha}(\frac{\sigma}{\sqrt{n}})
\\
\bar{x}=50 + (2.326347)(\frac{10}{\sqrt{50}})
\\
\bar{x}=53.289952714
$
Now we calculate the type II error
\
$
z=\frac{\bar{x} - \mu}{\frac{\sigma}{\sqrt{n}}}
\\
z=\frac{53.289952714 - 55}{\frac{10}{\sqrt{50}}}
\\
z=−1.20918
\\
$
\
Now we find the p-value of that z-score
\
1-0.113296=0.88670315
\
This is the power
# $\alpha=0.001$
First step is to set the hypothesis
\
$
H_0: \mu =50
\\
H_a: \mu > 50
$
\
In this question we are going to use a z-test
\
$
z = \frac{\sqrt{n}(\bar{x} - \mu_0)}{\sigma} = \frac{\sqrt{50}(55-50)}{10} = 3.535534$
\
Now we need to find the critical values
\
$
z_{1-\alpha} = z_{0.999} = 3.0902323$
\
Is $z > z_{0.99}$? Yes
\
Therefore we reject the null hypothesis
\
Now we are going to calculate the power
\
$
\bar{x}=\mu + z_{1-alpha}(\frac{\sigma}{\sqrt{n}})
\\
\bar{x}=50 + (3.090232)(\frac{10}{\sqrt{50}})
\\
\bar{x}=54.370248
$
Now we calculate the type II error
\
$
z=\frac{\bar{x} - \mu}{\frac{\sigma}{\sqrt{n}}}
\\
z=\frac{54.370248438 - 55}{\frac{10}{\sqrt{50}}}
\\
z=−0.4453016
\\
$
\
Now we find the p-value of that z-score
\
1-0.3280509=0.671949094
\
This is the power

In [0]:
# Part A
qnorm(0.95)
# Part B
pnorm(-1.8905)
# Part C
qnorm(0.99)
pnorm(-1.20918)
qnorm(0.999)
pnorm(-0.4453016)

[1] 1.644854

[1] 0.02934556

[1] 2.326348

[1] 0.1132968

[1] 3.090232

[1] 0.3280509

## 3. Assume that $X$ is normally distributed with unknown mean $\mu$ and standard deviation $\sigma = 16$. Let’s set the probability of committing a Type I error to be $\alpha = 0.05$. Let’s check $H0: \mu = 100$ against the alternative hypothesis that $H1: \mu > 100$. What is the power of the hypothesis test if the true population mean were:

### a. $\mu_0 = 108$?

### b. $\mu_0 = 112$?

### c. $\mu_0 = 116$?

### d. $\mu_0 = 108$ and $\alpha = 0.01$?


# Part A
Since we are given the standard deviation of the population we can use the z-test
\
$
\bar{x}=\mu + z_{1-alpha}(\frac{\sigma}{\sqrt{n}})
\\
\bar{x}=100 + (1.645)(\frac{16}{\sqrt{24}})
\\
\bar{x}=105.3725
$
\
Now we calculate the type II error
\
$
z=\frac{\bar{x} - \mu}{\frac{\sigma}{\sqrt{n}}}
\\
z=\frac{105.37254 - 108}{\frac{16}{\sqrt{24}}}
\\
z=−0.804489743
\\
$
\
Now we find the p-value of that z-score
\
1-0.210557095342901=0.78944
This is the power
\
# Part B
$
\bar{x}=\mu + z_{1-alpha}(\frac{\sigma}{\sqrt{n}})
\\
\bar{x}=100 + (1.645)(\frac{16}{\sqrt{24}})
\\
\bar{x}=105.3725
$
Now we calculate the type II error
\
$
z=\frac{\bar{x} - \mu}{\frac{\sigma}{\sqrt{n}}}
\\
z=\frac{105.3725 - 112}{\frac{16}{\sqrt{24}}}
\\
z=−2.02924
\\
$
\
Now we find the p-value of that z-score
\
1-0.0212169256156377=0.978783
This is the power
# Part C
$
\bar{x}=\mu + z_{1-alpha}(\frac{\sigma}{\sqrt{n}})
\\
\bar{x}=100 + (1.645)(\frac{16}{\sqrt{16}})
\\
\bar{x}=105.3725
$
Now we calculate the type II error
\
$
z=\frac{\bar{x} - \mu}{\frac{\sigma}{\sqrt{n}}}
\\
z=\frac{105.3725 - 116}{\frac{16}{\sqrt{24}}}
\\
z=−3.25399403
\\
$
\
Now we find the p-value of that z-score
\
1-0.000568973355=0.999431027
This is the power
# Part D
$
\bar{x}=\mu + z_{1-alpha}(\frac{\sigma}{\sqrt{n}})
\\
\bar{x}=100 + (2.3263)(\frac{16}{\sqrt{24}})
\\
\bar{x}=107.597663985
$
Now we calculate the type II error
\
$
z=\frac{\bar{x} - \mu}{\frac{\sigma}{\sqrt{n}}}
\\
z=\frac{107.597663985 - 108}{\frac{16}{\sqrt{24}}}
\\
z=−0.123189743
\\
$
\
Now we find the p-value of that z-score
\
1-0.45097842=0.549021576
This is the power


In [0]:
qnorm(1-0.01)
pnorm(-0.123189743)

[1] 2.326348

[1] 0.4509784

## 4. In healthy adult males, sciatic nerve conduction velocity (in cm/msec) is normally distributed ~ $\mathcal{N}(65, 25)$. The conduction velocities of $n = 8$ subjects admitted to the poison control center of a metropolitan hospital with a diagnosis of methylmercury poisoning had a mean conduction velocity of $55 cm/msec$ and a variance of $49 (cm/msec)^2$. 

### a. Do these data provide sufficient evidence to indicate that the conduction velocities are significantly slower in the poisoned individuals?

### b. Do these data provide sufficient evidence to indicate that poisoned individuals are more variable in their sciatic nerve conduction velocities than normal individuals?


a) In this question we are given the population mean and standard deviation and the standard deviation and mean of a sample and we are being asked to determine the level of significance of the sample.
$
\mu= 65
\\
\sigma=5
\\
n=8
\\
\bar{x}=55
\\
s^2=49
$

The first step is to set the two hypothesis
\
$
\\
H_0: \mu=65
\\
H_a: \mu < 65
$
\
Next we need to determine which stastical test to use
\
For this case, we are going to do a z-test since we are given the population standard deviation and the population mean
\
$
z=\frac{\bar{x}-\mu}{\frac{\sigma}{\sqrt{n}}}
$
\
$
z=\frac{55-65}{\frac{5}{\sqrt{8}}}=-5.656854
$
\
Then we find the p-value for this z-score
\
The p-value that we found is almost zero which is smaller than 0.05 which means that we reject the null hypothesis and therefore we accept the alternative hypothesis.
\
Therefore, this data provides significant evidence to indicate that the conduction velocities are significantly slower in the poisoned individuals


In [0]:
# Part A
pmean<-65
ps<-5
n<-8
xbar<-5.6568
pnorm(xbar,mean=pmean,sd=ps/sqrt(n))


[1] 2.331733e-247

B) The first step is to set the two hypothesis
\
$
\\
H_0: \sigma^2 \leq 25
\\
H_a: \sigma^2 > 25
$
\
Next we need to determine which stastical test to use
\
For this case, we are going to do a chi-square test since we know the population standard deviation
\
$
X^2=\frac{(n-1)s^2}{\sigma^2}
\\
X^2=\frac{(8-1)(49)}{25}
\\
X^2=13.72
$
\
Then we use R to find the p-value for the chi-square distribition

In [0]:
n<-8
ss2<- 49
o2<-25
x2<-((n-1)*ss2)/(o2)
pchisq(x2,(n-1),lower.tail = FALSE)

[1] 0.05639151

The p-value for this case is greater than the alpha value which means that we failed to reject the null hypothesis and therefore we can say that this data does not provide enough sufficient evidence to indicate that poisoned individuals are more variable in their sciatic nerve conduction velocities than normal individuals

## 5. The mean head circumference of newborn and one-year-old males in the United States is $35.83 cm$ and $46.35 cm$, respectively. The head circumferences of $n = 15$ males measured at birth who were later diagnosed with autism spectrum disorder are given below.

$(34.8, 35.0, 36.6, 35.9, 32.8, 33.4, 33.3, 35.0, 34.9, 35.2, 35.9, 34.9, 31.8, 34.6, 33.1)$

###  a. Is there evidence that head circumference at birth for these males was significantly smaller than normal?

### b. The same group measured a year later had $\overline{x} = 47.64 cm$ and $s = 1.19 cm$. Is there evidence that head circumference at one year for these males was significantly larger than normal?



# Part A
$
\bar{x}=34.48
\\
s=1.3235234
$
\
Now we have to set the hypothesis for this question
\
$
H_0: \mu=35.83
\\
H_1: \mu < 35.83
$
\
Since in this question they do not provided us with a population variance or standard deviation we have to use a t-test
\
$
t_{observed}= \frac{\bar{x} - \mu}{\frac{\sigma}{\sqrt{n}}}
\\
t_{observed}= \frac{34.48 - 35.83}{\frac{1.3235}{\sqrt{15}}}
\\
t_{observed}=−3.9505
\\
$
\
Now we use R to find the p-value for that t-statistics at a 14 degrees of freedom
\
The p-value for this t-test is 0.000725
\
Since this value is way smaller than 0.05 
\
We reject the null hypothesis and we accept the alternative hypothesis which is that the head circumference of this male was significantly smaller than normal.
 # Part B
 \
 $
H_0: \mu=46.35
\\
H_1: \mu > 46.35
$
\
Since in this question they do not provided us with a population variance or standard deviation we have to use a t-test
\
$
t_{observed}= \frac{\bar{x} - \mu}{\frac{\sigma}{\sqrt{n}}}
\\
t_{observed}= \frac{47.64 - 46.35}{\frac{1.19}{\sqrt{15}}}
\\
t_{observed}=4.1984
\\
$
\
Now we use R to find the p-value for that t-statistics at a 14 degrees of freedom
\
The p-value for this t-test is 0.000446
\
Since this value is way smaller than 0.05 
\
We reject the null hypothesis and we accept the alternative hypothesis which is that the head circumference of this male was significantly larger than normal.

In [0]:
x<-c(34.8,35.0,36.6,35.9,32.8,33.4,33.3,35.0,34.9,35.2,35.9,34.9,31.8,34.6,33.1)
mean(x)
sd(x)
pt(-3.9505,14)
pt(4.1984,14,lower.tail = FALSE)

[1] 34.48

[1] 1.323523

[1] 0.0007251672

[1] 0.0004466587

## 6. Both lifestyle and genetic factors play a role in longevity. Recent studies have linked large cholesterol particles to good health and longevity. A variant of a gene called CETP encoding the cholestryl ester transferase protein apparently causes the formation of large cholesterol particles. In a particular population, the life spans for males (in years) are normally distributed with $\mathcal{N}(74.2, 100.0)$. A sample of $n = 8$ males in this population that had the variant CETP gene lived an average of $81.2$ yr with a standard deviation of $8.0$ yr.

### a. Does this sample indicate people with the CETP variant gene live significantly longer than those without it?

### b. Does the sample indicate that the CETP variant carriers are significantly less variable in their life spans than the general population?


# Part A
The first step is to setup the hypothesis
\
$
H_0: \mu=74.2
\\
H_a: \mu > 74.2
$
\
Even though we are given the mean and standard deviation of the population we have the opportunity to us a z-test however since we have a really small number of samples we have to use a t-test
\
$
\alpha =0.05
\\
\bar{x}=81.2
\\
s=8
\\
n=8
\\
$
\
Now we need to use a t-test to find the p-value
\
$
t_{observed}=\frac{\bar{x}- \mu}{\frac{s}{\sqrt{n}}}
\\
t_{observed}=\frac{81.2-74.2}{\frac{8}{\sqrt{8}}}
\\
t_{observed}=2.4749
$
\
Now we use R to find the P-value of this t-test
\
The p-value for this t-test is 0.02126
\
Since this value is smaller than 0.05
\
We reject the null hypothesis and we accept the alternative hypothesis which means that people with the CETP variant gene live significantly longer than those without it
# Part B
The first step is to set the hypothesis
\
$
H_0: \sigma^2 \geq 100
\\
H_a: \sigma^2 < 100
$
\
For this problem, we are going to use a chi-square test
\
$
\alpha =0.05
\\
\bar{x}=81.2
\\
s^2=64
\\
n=8
\\
$
\
The first step is to find the critical value $X_{0.95}^2$
\
Since in this question the alternative hypothesis is for values less than the population variance we have to use a left tailed test for the chi-square distribution since the common cumulative chi square distribution is calculated via a right tailed method
\
Now we have to find the test statistics for the sample using the chi-square
\
$
X^2=\frac{(n-1)s^2}{\sigma^2}
\\
X^2=\frac{(8-1)64}{100}
\\
X^2=4.48
$
\
Since the test statistics is greater than the critical value, we fail to reject the null hypothesis. Therefore, this sample does not indicate that the CETP variant carriers are significantly less variable in their life spans than the general population

In [0]:
# Part A
pt(2.4749,7,lower.tail = FALSE)
# Part B
qchisq((1-0.95),7)

[1] 0.0212633

[1] 2.16735

##  7. A physical anthropologist studying the effects of nutritional changes on human morphometrics is interested in the impact of Western diet on the heights of adult Japanese males. Studies prior to World War II indicated that the heights of adult Japanese males had a mean of 64.0 inches and a standard deviation of 2.0 inches. The anthropologist measured the heights of a random sample of 50 adult Japanese men and found their height to average 66.5 inches with a standard deviation of 3.0 inches.

### a. Calculate the 95% confidence interval for mean from the sample.
### b. Calculate the 95% confidence interval for variance from the sample.
### c. Has the mean height changed since World War II? Write out the hypotheses associated with this question.
### d. Has the variance of heights changed since World War II? Again, write out the hypothesis associated with this question.
### e. Relate the confidence intervals in (a) and (b) to the hypotheses in (c) and (d).


# Part A
$
\mu = 64
\\
\sigma = 2.0
\\
n=50
\\
\bar{x}=66.5
\\
\bar{s}=3
$
\
Givent that we have a sample that is greater than 30 we can use a z-distribution to approximate the mean of the sample
\
$
\bar{X} \pm Z_{0.975} * \frac{s}{\sqrt{n}}
\\
66.5 \pm 1.96 * \frac{3}{\sqrt{50}}
\\
66.5 \pm 0.8315423
$

# Part B
To approximate the value of the variance we are going to use a chi-square distrubution
\
$$\begin{equation}
\frac{(n-1)s^2}{x^2_{\frac{\alpha}{2}}} \leq \sigma^2 \leq \frac{(n-1)s^2}{x^2_{1-\frac{\alpha}{2}}}
\\
\frac{(50-1)3^2}{x^2_{\frac{\alpha}{2}}} \leq \sigma^2 \leq \frac{(50-1)3^2}{x^2_{1-\frac{\alpha}{2}}}
\\
\frac{441}{x^2_{\frac{\alpha}{2}}} \leq \sigma^2 \leq \frac{441}{x^2_{1-\frac{\alpha}{2}}}
\\
\frac{441}{70.22245} \leq \sigma^2 \leq \frac{441}{31.55671}
\\
6.28004 \leq \sigma^2 \leq 13.9748
\\
\end{equation}$$

# Part C
$
H_0=64
\\
H_a \ne 64
\\
$

# Part D
$
H_0=4
\\
H_a \ne 4
\\
$

# Part E

Since confidence interval of part a does not contain 64 so null hypothesis of part c is rejected.
\
Since confidence interval of part b does not contain 4 so null hypothesis of part d is rejected.

In [0]:
#Part A
# Confidence Interval for the population mean
sm<-66.5
sstd<-3
n<-50
error<-qnorm(0.975) * sstd/sqrt(n)
print('Confidence Interval for mean of the population')
sm + error 
sm - error
#Part B
# Confidence Interval for the Variance mean
print('Confidence Interval for variance of the population')
((n-1)*(sstd)^2)/qchisq(0.025,df=49, lower.tail=FALSE)
((n-1)*(sstd)^2)/qchisq(0.975,df=49, lower.tail=FALSE)

[1] "Confidence Interval for mean of the population"


[1] 67.33154

[1] 65.66846

[1] "Confidence Interval for variance of the population"


[1] 6.280046

[1] 13.97564

## 8. A potential side effect of using oral contraceptives is an increase in blood pressure. Given below are the systolic blood pressures of 10 women measured before beginning and after having taken an oral contraceptive for a 6-month period. 

> Woman | Before | After |
> :---: | :---:  | :---: | 
>   1   |  113   |  118  |
>   2   |  117   |  123  |
>   3   |  111   |  114  |
>   4   |  107   |  115  |
>   5   |  115   |  122  |
>   6   |  134   |  140  |
>   7   |  121   |  120  |
>   8   |  108   |  105  |
>   9   |  106   |  111  |
>   10  |  125   |  129  |

### Assuming systolic blood pressure to be normally distributed, do these data suggest that using an oral contraceptive significantly increases systolic pressure?

Step 1: Set the hypothesis
$
H_0:\mu = 0
\\
H_a:\mu >0
\\
\\
\alpha =0.05
$
\
Step 2
\
Calculate the mean, standard deviation, and standard error of the difference between the two values
\

In [0]:
x1<-c(113,117,111,107,115,134,121,108,106,125)
x2<-c(118,123,114,115,122,140,120,105,111,129)
d<-c(x1 - x2)
mean(d)
sd(d)
sd(d)/sqrt(length(d))

[1] -4

[1] 3.496029

[1] 1.105542

Step 3
\
Calculate the test statistics
$
t=\frac{\bar{d}}{\frac{s}{\sqrt{n}}}
\\
t=\frac{-4}{\frac{3.49603}{\sqrt{10}}}
\\
t=-3.62
$
\
Now we need to find the P-value using R

In [0]:
pt(-3.62,9)

[1] 0.002785877

Since the p-value is smaller than $\alpha$=0.05
\
We reject the null hypothesis and we accept the alternative hypothesis which means that there is a significant difference between the two means

## 9. Suppose that you mistakenly believed that the data sets from #8 were independent samples taken from two different populations: (i) women not taking contraceptives and (ii) women taking contraceptives. 

### a. Which test would now be appropriate for examining whether there's a significant difference between the two populations? 

### b. Carry this test out and determine whether you would have found that women taking contraceptives have a significantly higher systolic pressure. 

### c. Explain the cause of any differences in your results when compared to the previous problem.


# Part A

The easiest test to determine whether there is a significant differences between the two populations is an F-test
# Part B
$
\bar{X_{before}}=115.7
\\
\bar{X_{while}}=119.7
\\
S_{1}^2=78.9
\\
S_{2}^2=96.01
$
\
The first step is to set the hypothesis for the f-test since we are testing if the difference between the two variances come from the same population
\
$
H_0: \sigma_1 ^2 = \sigma_1 ^2
\\
H_a:  \sigma_1 ^2 \ne \sigma_1 ^2
$
\
F-test statistics
\
$
F=\frac{s_1 ^2}{s_2 ^2}
\\
F=\frac{78.90}{96.01}
\\
F=0.82
$
\
Now we use R to find the critical value for the f test
\
The critical value for the f-test is between 4.03 and 0.2483
\
Since 0.82 is between this two values it means that we fail to reject the null hypothesis and therefore we can say that there is no significant difference between the two population variances
\
In other words, the population variances for the two sample populations come from the same population
\
Now that we know that the samples come from the same population we can use t-statistics
\
Before we do that, we have to do set the hypothesis for this test
\
$
H_0: \mu_1 = \mu_2
\\
H_a: \mu_1 < \mu_2
$
\
For this question we have to find pooled standard deviation since we the two sample come from the same population
\
$
S_p=\sqrt{\frac{(n_1-1)s_1 ^2 + (n_2-1)s_2 ^2}{n_1 + n_2 - 2}}
\\
S_p={\frac{(10-1)*78.90 ^2 + (10-1)* 96.01 ^2}{10 + 10 - 2}}
\\
S_p=87.455
$
\
Now we use the t-test for two samples
\
$
t=\frac{(\bar{x_1} - \bar{x_2})-(\mu_1 - \mu_2)}{s_p * \sqrt{\frac{1}{n_1}+\frac{1}{n_2}}}
\\
t=\frac{(115.7 - 119.7)-(0)}{87.455 * \sqrt{\frac{1}{10}+\frac{1}{10}}}
\\
t=-0.10
$
\
Now we use R to find the p-value given the t statistics
\
The p-value is 0.46072 which is way above the alpha of 0.05
\
Therefore we fail to reject the null hypothesis and there is no difference between the two means
# Part C
In problem 8, we said that there is a significant difference between the two means while in this test we said that there was no significant difference. This is due to problem 8 assuming they are independent events which makes the probability to be more probable since it doesn't account for the intersection between the two populations like in problem 9


In [0]:
x1<-c(113,117,111,107,115,134,121,108,106,125)
x2<-c(118,123,114,115,122,140,120,105,111,129)
mean(x1)
var(x1)
mean(x2)
var(x2)
qf(0.05/2,9,9, lower.tail = FALSE)
qf(0.05/2,9,9, lower.tail = TRUE)
pt(-0.10,18)

[1] 115.7

[1] 78.9

[1] 119.7

[1] 96.01111

[1] 4.025994

[1] 0.2483859

[1] 0.4607248

## 10. A biochemist studying the activity of the enzyme triose phosphate isomerase (TPI) in a species of Drosophila wants to know if the mean activity of this enzyme is the same at $pH = 5$ and $pH = 8$. 

> Enzyme activity ($\frac{\mu M}{min}$) | | | | | |
> :---: | :---:  | :---: | :---: | :---:  | :---: 
> $\textbf{pH = 5}$ | 11.1 | 10.0 | 13.3 | 10.5 | 11.3
> $\textbf{pH = 8}$ | 12.0 | 15.3 | 15.1 | 15.0 | 13.2

### a. Given the above data and assuming enzyme activities are normally distributed, determine if the mean activities of TPI are significantly different at these two pH levels.

### b. Give the 95% confidence interval for $\mu_5 - \mu_8$ .

a)
\
Step 1: Determine the hypothesis that we are going to be testing
\
$
H_0:\mu_1 - \mu_2 \geq 0
\\
H_a:\mu_1 - \mu_2 \leq 0
$
\
\
Step 2: Determine which test you are going to use
\
We are going to do an independent sample t-test
\
Ph= 5 Sample
\
$
\bar{x_1}= 11.24
\\
s_1=1.26
\\
n_1=5
$
\
\
Ph= 8 Sample
\
$
\bar{x_2}= 14.12
\\
s_2=1.45
\\
n_2=5
$
\
\
Significance level= $\alpha=0.05$
\
Standard Error
\
$\sigma_{\bar{x_1} - \bar{x_2}}=\sqrt{\frac{s_1^2}{n_1}+ \frac{s_2^2}{n_2}}$
\
$\sigma_{\bar{x_1} - \bar{x_2}}=\sqrt{\frac{1.26^2}{5}+ \frac{1.45^2}{5}}$
\
$\sigma_{\bar{x_1} - \bar{x_2}}=0.8591$
\
T test
\
$t_{observed}=\frac{\bar{x_1}- \bar{x_2}}{\sqrt{\frac{s_1^2}{n_1}+ \frac{s_2^2}{n_2}}}
\\
t_{observed}=\frac{11.24- 14.12}{0.8591}
\\
t_{observed}=-3.35242
$
Now we use R to find the p-value

In [0]:
x<-(11.24-14.12)/0.8590809
pt(x,4)

[1] 0.01425235

Now we calculate the critical values

In [0]:
qt(0.975,4, lower.tail = F )

[1] -2.776445

Now we compare our results
\
$
T_{observed}<T_{critical}
\\
−3.35242 < -2.77644510519779
$
\
Therefore we reject the null hypothesis and accept the alternative hypothesis
Which means that the difference between the two means is signifcant

Part B
\
$
\bar{X_1}-\bar{X_2}\pm t_{0.05,8} (0.9624)
\\
11.24 -14.12\pm (2.306)*(0.9624)
\\
-2.88\pm 2.2193
$


## 11. Problem 2 continued … You collected some data and got this for the number of days that the battery lasted in each sample:
    
$(491, 485, 503, 492, 482, 490, 489, 495, 497, 487, 493, 480, 482, 504, 501, 486, 478, 492, 482, 502, 485, 503, 497, 500, 488, 475, 478, 490, 487, 486)$
    
### Find the sample statistic, test statistic, and p-value, and interpret your results.


**Sample Statistics**
\
Mean=490
\
Standard Deviation=8.20008
\
**Test Statistics**
\
For this question we are going to use a t-test since the number of samples is small
\
Hypothesis
\
$
H_0:\mu \geq 500
\\
H_a:\mu<500
$
\
Critical Values for the test statistics
\
$
t_{n-1, 1-\alpha} = t_{29, 0.05} = -1.69912
$
\
T-statistics
\
$t = \frac{\sqrt{n}(\overline{x} - \mu_0)}{s} = \frac{\sqrt{30}(490 - 500)}{8.20008} = −6.67954$
\
Based on this T-value we can say that we reject the null hypothesis
\
And therefore battery life is significantly lower than the one specified by the manufacturer
\
**P-value**
For the previous t-test the p-value is 1.25794362036123e-07
which is super small which means that the baterry is significantly lower than the one specified by the manufacturer

In [0]:
x<-c(491,485,503,492,482,490,489,495,497,487,493,480,482,504,501,486,478,492,482,502,485,503,497,500,488,475,478,490,487,486)
mean(x)
sd(x)
length(x)
qt(0.05, df=29)
pt(-6.67954,29)

[1] 490

[1] 8.200084

[1] 30

[1] -1.699127

[1] 1.257944e-07